<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDevicePreview</class>
 <widget class="QWidget" name="CDevicePreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>29</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_content2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>content</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_content1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>114</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>content</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
